src/utils/createDurationString.ts

```diff
@@ -0,0 +1,26 @@
+import pad from './pad';
+
+const createDurationString = (duration: number) => {
```

Comment:
Isn't there already a formatTimestamp function somewhere that does this?

Reply:
I'll double check that!